code/game/objects/items/robot/robot_items/robot_rack.dm

```diff
@@ -127,4 +127,22 @@
 /obj/item/robot_rack/ammo/a12mm/update_icon()
 	icon_state = "ammopack_[length(held)]"
 
-#undef NEEDED_CHARGE_TO_RESTOCK_MAG
\ No newline at end of file
+#undef NEEDED_CHARGE_TO_RESTOCK_MAG
+
+/obj/item/robot_rack/atmos
+	name = "portable atmospherics rack"
+	desc = "Used by engineering cyborgs for convenient transport of portable atmospherics machinery."
+	icon = 'icons/obj/rollerbed.dmi'
+	icon_state = "borgbed_deployed"
```

Comment:
icon

Reply:
Yes, it's the deployed icon from bed racks.